src/data/queries/tests/accordion.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { ParagraphAccordion } from '@/types/drupal/paragraph'
+import { queries } from '@/data/queries'
+import mockData from '@/mocks/Accordion.mock.json'
```

Comment:
this fails because uppercase to lowercase. i need to fix this in the generated template, but can you lowercase it here for now?